media-libs/mlt/mlt-6.18.0.recipe

```diff
@@ -0,0 +1,66 @@
+SUMMARY="An open source multimedia framework"
+DESCRIPTION="It provides a toolkit for video editors and many more applications"
+HOMEPAGE="https://www.mltframework.org/"
+COPYRIGHT="2011-2018 by Meltytech, LLC."
+LICENSE="GNU LGPL v2.1"
+REVISION="1"
+SOURCE_URI="https://github.com/mltframework/mlt/releases/download/v$portVersion/mlt-$portVersion.tar.gz"
+CHECKSUM_SHA256="fd2a346e0cc093e5d813e04cd126c077a2d224ae4d17b5226ff7478feaa445ea"
+SOURCE_DIR="mlt-$portVersion"
+PATCHES="mlt-$portVersion.patchset"
+
+ARCHITECTURES="x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	mlt$secondaryArchSuffix = $portVersion
+	cmd:melt
+	lib:libmlt$secondaryArchSuffix
+	lib:libmlt++$secondaryArchSuffix
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	mlt${secondaryArchSuffix}_devel = $portVersion
+	devel:libmlt$secondaryArchSuffix
+	devel:libmlt++$secondaryArchSuffix
+	"
+REQUIRES_devel="
+	mlt$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:awk
+	cmd:find
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	runConfigure ./configure
+	make PREFIX=$prefix -j$jobs
+}
+
+INSTALL()
+{
+	make install PREFIX=$prefix
+
+	mv $prefix/include/* $includeDir
+	rm -rf $prefix/include
+
+	prepareInstalledDevelLibs libmlt
+	prepareInstalledDevelLibs libmlt++
```

Comment:
Call prepareInstalledDevelLibs only once

Reply:
After modifying this point, the build still does not work.